tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -253,10 +220,37 @@ namespace.gluster:
         pool:
           help: The underlying pool name of the brick
           type: String
+        vol_id:
+          help: Volume id
+          type: String
+        used:
+          help: whether the brick is part of a volume
+          type: Boolean
+        sequence_number:
+          help: Sequence number amongst gluster replica-set
+          type: Integer
+        hostname:
+          help: Name of the host
+          type: String
+        port:
+          help: Brick port
+          type: int
+        status:
+          help: Status of the brick
+          type: String
+        filesystem_type:
+          help: File system type
+          type: String
+        mount_opts:
+          help: Mount options
+          type: String
+        utilization:
+          help: Brick utilization
+          type: Dict        
```

Comment:
Is this a simple dict or a nested dict? If it is a nested dict, make the type as "json"